accessibility/.pa11yci-mobile.json

```diff
@@ -0,0 +1,23 @@
+{
+  "defaults": {
+    "timeout": 240000,
+    "runners": ["axe"],
+    "ignore": ["color-contrast"],
+    "concurrency": 1,
+    "chromeLaunchConfig": {
+      "ignoreHTTPSErrors": true,
+      "args": ["--disable-dev-shm-usage", "--no-sandbox"]
+    },
+    "viewport": {
+      "width": 390,
+      "height": 844,
```

Comment:
any idea where we got these numbers?

Reply:
I think originally from the DOJ project https://github.com/usdoj-crt/beta-ada/blob/main/.pa11yci-mobile

Looks like it's the iphone 12 dimensions chrome device emulator